jest.js

```diff
@@ -0,0 +1,15 @@
+const pluginJest = require('eslint-plugin-jest');
+
+// Configuration for Jest test files only.
+module.exports = [
+    {
+        files: ['**/*.spec.{js,ts}', '**/*.test.{js,ts}'],
```

Comment:
I think this should be expanded to also match `**/test/**/*.{js,ts}` or `**/tests/**/*.{js,ts}`, as we often have some shared util files in tests which use the Jest globals like `each()`.

Reply:
yes, good point